packages/tools/pluggable-widgets-tools/scripts/toggle-csp.mjs

```diff
@@ -0,0 +1,123 @@
+import sh from "shelljs";
+import { join, dirname } from "path";
+import { fileURLToPath } from "url";
+import dotenv from "dotenv";
+
+const log = {
+    info: (...args) => console.info(...args),
+    error: (...args) => {
+        console.error("Ooops! We have a problem:", ...args);
+    }
+};
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const cspFilesDir = join(__dirname, "csp-files");
+
+function $file(p) {
+    return join(sh.env.MX_PROJECT_PATH, p);
+}
+
+function silent(operation) {
+    const silent = sh.config.silent;
+    sh.config.silent = true;
+    operation();
+    sh.config.silent = silent;
+}
+
+function themeWebListing() {
+    const dir = $file("theme/web");
+    log.info(dir, "contnet:");
+    log.info(sh.ls(dir).stdout);
+}
+
+function done() {
+    log.info("Done.", "\n");
+    themeWebListing();
+}
+
+function copy() {
+    log.info("Copy CSP boilerplate...");
+    sh.cp(join(cspFilesDir, "*"), $file("theme/web"));
+}
+
+function backup() {
+    const backup = $file("theme/web/index.backup.html");
+    const index = $file("theme/web/index.html");
+    if (sh.test("-f", index)) {
+        log.info("Making backup for index.html...");
```

Comment:
Why we backup and restore only this file, but not other files we possibly replace?